src/main/java/com/pirimid/cryptotrade/DTO/PlaceOrderReqDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.pirimid.cryptotrade.DTO;
+
+import lombok.*;
```

Comment:
Can you remove * ? Do check in other file also.

Reply:
We try to add explicitly imports for Getter, Setter, NoArgsConstuctor, AllArgsConstructor  but after adding all these IntelliJ is automatically converting this to ".*".

Reply:
you can add this setting in intellij